docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
+    togglesidebar.onkeydown = handleEnterKey;
```

Comment:
^^ Also being said, this code here is in docs.js which probably will never get react, in which case we'd have to keep using semantic ui JS, and so maybe Jquery?